src/test/java/tests/TurboLabelTest.java

```diff
@@ -71,5 +74,26 @@ public void turboLabelTest() {
         assertEquals("dummy/dummy", turboLabel.getRepoId());
 
     }
+    
+    @Test
+    public void matchedLabel() {
```

Comment:
What is being tested here? Name the method to reflect method being tested. In addition, add comments to explain test scenario. Tests often requires more comments than functional code because we need to explain the rationale for the test case.